src/logger.js

```diff
@@ -0,0 +1,10 @@
+const bunyan = require('bunyan')
+
+const defaultOptions = {
+  codeVersion: process.env.CODE_VERSION,
+  serializers: {
```

Comment:
Are we sure this is needed? We might actually be removing the default serializers other than `err` with this.

Reply:
Indeed, I'll revert this